buildstockbatch/aws/aws.py

```diff
@@ -1723,7 +1723,7 @@ def run_job(cls, job_id, bucket, prefix, job_name, region):
                     logger.debug("Extracting {}".format(epw_filename))
                     f_out.write(gzip.decompress(f_gz.getvalue()))
 
-        cls.run_simulations(cfg, jobs_d, job_id, sim_dir, S3FileSystem(), bucket, prefix)
+        cls.run_simulations(cfg, jobs_d, job_id, sim_dir, S3FileSystem(), f"{bucket}/{prefix}")
```

Comment:
I see that in the `run_simulations` function definition there's just an `output_path` argument. Does that mean this wasn't already working?

Reply:
I changed the function params in response to [this comment](https://github.com/NREL/buildstockbatch/pull/422#discussion_r1414573255), but missed updating this line.

Meanwhile, I'm also looking into why the test didn't catch this.

Reply:
Added some comments to test_docker_base.py to at least explain why the test there won't catch issues like the one in docker_base.py.

We should try to get that test to test things more thoroughly, but that's more involved so I'll save it for a separate PR.